timesketch/frontend/src/utils/RestApiClient.js

```diff
@@ -101,6 +101,18 @@ export default {
     }
     return RestApiBlobClient.post('/sketches/' + sketchId + '/archive/', formData)
   },
+  getSketchAttributes (sketchId) {
+    return RestApiClient.get('/sketches/' + sketchId + '/attribute/')
```

Comment:
I would prefer to make the naming consistent with other API resources. Make this `/attributes/` instead. Let's not do that in this PR though, but lets followup with this. :)

Reply:
Yeah, let's do the API changes in another PR. I'll update this client at the same time.

Reply:
SG